Spiegel Online.js

```diff
@@ -39,64 +39,43 @@ http://www.spiegel.de/international/europe/0,1518,700530,00.html
 */
 
 function detectWeb(doc, url) {
-
-	var spiegel_article_XPath = './/div[@class="column-both"]/h2[@class="article-title"]|.//div[@class="column-wide"]/h2[contains(@class, "headline")]';
-	//the print edition is a magazine. Since the online edition is updated constantly it
-	//makes sense to treat it like a newspaper.
-	if (url.match(/\/print\//) && ZU.xpathText(doc, spiegel_article_XPath)){
-		return "magazineArticle";
-	}
-	else if (doc.evaluate(spiegel_article_XPath, doc, null, XPathResult.ANY_TYPE, null).iterateNext() ){ 
-		//Zotero.debug("newspaperArticle");
-		return "newspaperArticle";
-	} else if (doc.location.href.match(/^https?\:\/\/www\.spiegel\.de\/thema/)){ 
-		//Zotero.debug("multiple");
-		return "multiple";
-	}  else if (doc.location.href.match(/^https?\:\/\/www\.spiegel\.de\/suche/)){ 
-		//Zotero.debug("multiple");
-		return "multiple";
-	}  else if (doc.location.href.match(/^https?\:\/\/www\.spiegel\.de\/international\/search/)){ 
-		//Zotero.debug("multiple");
-		return "multiple";
-	} else if (doc.location.href.match(/^https?\:\/\/www\.spiegel\.de\/international\/topic/)){ 
-		//Zotero.debug("multiple");
+	var spiegel_article_XPath = '//h2[@class="article-title"]';
+	if (doc.location.href.match(/^https?\:\/\/www\.spiegel\.de\/(thema|suche|international\/search|international\/topic)\// )){ 
 		return "multiple";
-	} 
+	} else if (ZU.xpathText(doc, spiegel_article_XPath)) {
+		//the print edition is a magazine. Since the online edition is updated constantly it
+		//makes sense to treat it like a newspaper.
+		if (url.match(/\/print\//)) {
+			return "magazineArticle";
+		} else { 
+			return "newspaperArticle";
+		}
+	}
 }
 
 function scrape(doc, url) {
-	
- 	if (detectWeb(doc, url)=="magazineArticle") {
- 			var newItem = new Zotero.Item("magazineArticle");
- 	}
- 	else{
-		var newItem = new Zotero.Item("newspaperArticle");
- 	}
+	var newItem = new Zotero.Item(detectWeb(doc, url));
+
 	newItem.url = doc.location.href; 
 
 	// This is for the title 
-	
-	var title_xPath = '//div[@class="column-wide"]/h2[contains(@class, "headline")]';
-	if (doc.evaluate(title_xPath, doc, null, XPathResult.ANY_TYPE, null).iterateNext() ){ 
-		var title = doc.evaluate(title_xPath, doc, null, XPathResult.ANY_TYPE, null).iterateNext().textContent;
-		newItem.title = title;
-	} else if (ZU.xpathText(doc, '//div[@id="spArticleColumn"]/h2')) {
-		newItem.title = ZU.xpathText(doc, '//div[@id="spArticleColumn"]/h2');
-	} else {
-		var title = doc.evaluate('//title', doc, null, XPathResult.ANY_TYPE, null).iterateNext().textContent;
-		title = title.split(" - ")[0];
-		newItem.title = title;
-	}
+	newItem.title = ZU.xpathText(doc, '//h2[@class="article-title"]') 
+					|| ZU.xpathText(doc, '//meta[@property="og:title"]/@content') 
+					|| ZU.xpathText(doc, '//title');
+	newItem.title = ZU.trimInternal(newItem.title);
```

Comment:
nit: use single tab to indent (tab widths differ for different setups, so trying to align with tabs is rather pointless)